fbw-a380x/src/systems/instruments/src/MFD/FMC/FmcAircraftInterface.tsx

```diff
@@ -1171,7 +1171,7 @@ export class FmcAircraftInterface {
     }
   }
 
-  private cruiseFlightLevelTimeOut: number | null = null;
+  private cruiseFlightLevelTimeOut: NodeJS.Timeout | null = null;
```

Comment:
```suggestion
  private cruiseFlightLevelTimeOut: ReturnType<typeof setTimeout> | null = null;
```
You can avoid the NodeJS type.

Reply:
very cool, thanks!